index.html

```diff
@@ -10,7 +10,7 @@
 <body>
     <header>   
             <nav> 
-                <img class="logo" src="" alt="">
+                <img class="un-k" src="" alt="">
                    <ul class="first-li">
                        <li><a href=""><img src="./images/facebook.png" alt=""></a></li>
                        <li><a href=""><img src="./images/twitter.png" alt=""></a></li> 
```

Comment:
- [ ] kindly add a hyperlink to your logo  that will redirect to home and also add the image attributes